internal/pkg/service/buffer/model/model.go

```diff
@@ -44,3 +44,11 @@ type Export struct {
 	Name             string            `json:"name" validate:"required,min=1,max=40"`
 	ImportConditions []ImportCondition `json:"importConditions" validate:"required"`
 }
+
+type Record struct {
```

Comment:
toto by mohol byt skor nejaky `RecordKey`, hladal som kde to ma content, a ako s tym pracujes.

Reply:
Aha, mas funkciu `RecordGet`, mozno by to mohlo byt `RecordKey.String()`, neviem

Reply:
Cože? Nerozumím. 😄 (Pokud by se to mělo použít jen jako parametr `configstore.CreateRecord()`, tak bych to asi přesunul tam. Ale předpokládal jsem že se to nejspíš využije i jinde. Nebo ne?)